test/e2e/specs/all-features/navbar/base-config.spec.js

```diff
@@ -44,52 +44,59 @@ describe('Navbar ', function() {
     });
 
     it('for the menu, should generate the correct # of list items based on acls to show/hide specific options', function() {
-        var nodesInDOM = menu.all(by.tagName('li'));
+        const nodesInDOM = menu.all(by.css('a'));
+        const headers = menu.all(by.css('.chaise-dropdown-header'));
         // Count the number of nodes that are being shown (top level and submenus)
         //   - Local: config has 13 but 1 is hidden by ACLs
         //   - CI: config has 13 but 7 are hidden based on ACLs
         var counter = (!process.env.CI ? 12 : 6); // counted from chaise config doc rather than having code count
+        let totalNodes = 0;
 
         nodesInDOM.count().then(function(count) {
-            expect(count).toEqual(counter, "number of nodes present does not match what's defined in chaise-config");
+            totalNodes += count;
+        });
+        headers.count().then(function(count) {
+            totalNodes += count;
+            expect(totalNodes).toEqual(counter, "number of nodes present does not match what's defined in chaise-config");
         });
     });
 
     it('should prefer markdownName over name when both are defined', function () {
         // option #2 has both name and markdownName defined
-        expect(element.all(by.css('#navbar-menu > li.dropdown')).get(1).getText()).toBe("Test Recordsets", "name was used instead of markdownName");
+        let secondMenuItem = menu.all(by.css('div.nav-item'));
+        expect(secondMenuItem.get(0).getText()).toBe("Test Recordsets", "name was used instead of markdownName");
     });
 
     it('should render a markdown pattern using proper HTML', function () {
         // in ci we don't have the same globus groups so the "show" ACL hides the 3rd link ("Records")
         var idx = (!process.env.CI ? 3 : 2);
         // option #4 has only markdownName defined
-        element.all(by.css('#navbar-menu > li.dropdown')).get(idx).element(by.css("a")).getAttribute('innerHTML').then(function (aInnerHTML) {
+        let lastMenuItem = menu.all(by.css('div.nav-item')).get(1);
```

Comment:
Thinking about this case more, since we are specifically trying to test the "2nd dropdown's displayed value", maybe it makes sense to make this selector more specific. The proposed change removes the need for `idx` since the 1st and 3rd links don't have the `nav-item` class.